tests/integrations/command/command.py

```diff
@@ -95,6 +95,10 @@ def get_command(id: str) -> Dict[str, Any]:
 def get_command_config(command_type: str, id: str) -> str:
     assert command_type in ["command", "notebook", "shell"]
     command = ["det", "-m", conf.make_master_url(), command_type, "config", id]
-    completed_process = subprocess.run(command, universal_newlines=True, stdout=subprocess.PIPE)
-    assert completed_process.returncode == 0
+    completed_process = subprocess.run(
+        command, universal_newlines=True, capture_output=True, env={"DET_DEBUG": "true"}
```

Comment:
`DET_DEBUG` is ridiculously verbose, I don't think we should be setting it here

Reply:
DET_DEBUG only tells the CLI to print out the full stack trace on a CLI failure. This does not effect the logging level.